sdk/package/manta-config.json

```diff
@@ -0,0 +1,114 @@
+{
+    "VERSION": "1.0.0",
+    "MIN_REQUIRED_SIGNER_VERSION": "0.7.0",
+    "SIGNER_URL": "http://localhost:29987/",
+    "NETWORKS": {
+        "Dolphin": {
+            "name": "Dolphin",
+            "ws": "wss://ws.rococo.dolphin.engineering",
+            "ws_local": "ws://127.0.0.1:9944",
+            "parachainId": 2084,
+            "decimals": 18,
+            "subscanUrl": "https://dolphin.subscan.io"
+        },
+        "Calamari": {
+            "name": "Calamari",
+            "ws": "wss://ws.calamari.systems/",
+            "ws_local": "ws://127.0.0.1:9944",
+            "parachainId": 2084,
+            "decimals": 12,
+            "subscanUrl": "https://calamari.subscan.io"
+        },
+        "Manta": {
+            "name": "Manta",
+            "ws": null,
+            "ws_local": null,
+            "parachainId": null,
+            "decimals": null,
+            "subscanUrl": null
+        }
+    },
+    "CALAMARI_PRIVATE_SS58_ADDRESS": "dmwQify37LiZkBEYP4jRd1BGDqxGb1Zax1gKUi3kQxCyaPwdF",
```

Comment:
This is the public address of the private ledger account on calamari, I just push a change to the name to avoid confusion. However, @ferrell-code is this address different on each network? If so then maybe we should move it into the "NETWORKS" section of the config.

Reply:
They are the same account on all networks. (public key: 0x6d6f646c6d616e74617061790000000000000000000000000000000000000000)

However these accounts can be represented by different SS58 encodings. Dolphin and calamari use the same SS58 encoding (78), so this address is correct for both. 

Manta runtime will use a different ss58 encoding but that is a future problem I suppose

Reply:
noted, will replace it with the public key and move the ss58 encoding into the NETWORKS section